content/en/docs/concepts/storage/projected-volumes.md

```diff
@@ -100,6 +100,8 @@ into their own volume mount outside of `C:\`.
 By default, the projected files will have the following ownership as shown for
 an example projected volume file:
 ```powershell
```

Comment:
BTW, this isn't PowerShell code.

Reply:
I thought adding the PowerShell command would make it appropriate 🤦🏽‍♂️ . Sorry about that. I will fix it if I am modifying this area again.